src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -73,7 +73,7 @@ internal static IArgument CreateArgumentOperation(ArgumentKind kind, IParameterS
             //      Note this check doesn't cover all scenarios. For example, when a parameter is a generic type but the type of the type argument 
             //      is undefined.
             if ((object)optionalParametersMethod == null 
-                || boundNode.HasAnyErrors
+                || hasErrors
```

Comment:
It looks like callers are now passing `boundNode.HasErrors` rather than `boundNode.HasAnyErrors`. I'm not sure what the implications are but was that change intentional?